app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -19,213 +20,201 @@
   </div>
 
   {{#if this.data.event.isSessionsSpeakersEnabled}}
-    <br>
-    <h3 class="ui dividing header">
-      <i class="ordered list icon"></i>
-      <div class="content">
-        {{t 'Sessions, Microlocations, Session Types'}}
-      </div>
-    </h3>
-    <div class="field">
-      <label class="required">Tracks</label>
-      {{#each this.tracks as |track|}}
-        <div class="{{if this.device.isMobile 'grouped'}} fields">
-          <div class="{{unless this.device.isMobile 'three wide'}} field">
-            <Input @type="text" @name="track" @value={{track.name}} placeholder="Name" />
-          </div>
-          <div class="{{unless this.device.isMobile 'four wide'}} field">
-            <Widgets::Forms::ColorPicker @value={{track.color}} @fontColor={{track.fontColor}}>
-              {{#if (gt this.tracks.length 1)}}
-                <button class="ui icon red button" type="button" {{action 'removeItem' track}}>
-                  <i class="minus icon"></i>
-                </button>
-              {{/if}}
-              <button class="ui icon primary button" type="button" {{action 'addItem' 'track'}}>
-                <i class="plus icon"></i>
-              </button>
-            </Widgets::Forms::ColorPicker>
-          </div>
-          <div class="{{unless this.device.isMobile 'two wide'}} field">
-            <input title="Preview"
-                   class="preview"
-                   value="{{if track.name track.name (t 'Preview Text')}}"
-                   readonly
-                   style={{css background-color=track.color color=(text-color track.color)}}>
-          </div>
-        </div>
-      {{/each}}
-    </div>
-
-    <div class="field">
-      <label class="required">{{t 'Microlocations or Virtual Online Rooms'}}</label>
-      {{#each this.microlocations as |microlocation|}}
-        <div class="{{if this.device.isMobile 'grouped'}} fields">
-          <div class="{{unless this.device.isMobile 'three wide'}} field">
-            <Input @type="text" @value={{microlocation.name}} placeholder="Name" />
-          </div>
-          <div class="{{unless this.device.isMobile 'four wide'}} field">
-            <div class="ui action input fluid">
-              <Input @type="number" @value={{microlocation.floor}} placeholder="Floor" />
-              {{#if (gt this.microlocations.length 1)}}
-                <button class="ui icon red button" type="button" {{action 'removeItem' microlocation}}>
-                  <i class="minus icon"></i>
-                </button>
-              {{/if}}
-              <button class="ui icon primary button" type="button" {{action 'addItem' 'microlocation'}}>
-                <i class="plus icon"></i>
-              </button>
-            </div>
-          </div>
-        </div>
-      {{/each}}
+  <br>
```

Comment:
Please don't change indentation unnecessarily. I can't see what's your change and what's just indentation change. Revert to original indentation and just make your changes in the PR

Reply:
Sorry for that, actually i use Prettier and as soon as i saved the changes it formatted everything. I will make a new PR.

Reply:
No need for a new PR, push changes to this PR itself